src/config/dynamic-theme-fixes.config

```diff
@@ -14212,6 +14212,9 @@ CSS
 body {
     background-color: var(--darkreader-neutral-background) !important;
 }
+body * {
+    caret-color: #898989 !important;
```

Comment:
Is it possible to use `${lightgray}`? Or any other fix that uses the following form: `${...}`

Reply:
```suggestion
    caret-color: ${gray} !important;
```